runtime/error.c

```diff
@@ -181,3 +187,83 @@ TaggedPtr BAL_LANG_ERROR_NAME(message)(TaggedPtr error) {
     ErrorPtr ep = taggedToPtr(error);
     return ep->message;
 }
+
+void _bal_print_mangled_name(const char *mangledName, FILE *fp) {
```

Comment:
This needs to written so that it has well-defined behaviour when passed arbitrary function names.

Reply:
Changed